src/main/host/process.rs

```diff
@@ -545,6 +544,11 @@ impl Process {
     }
 
     pub fn stop(&self, host: &Host) {
+        if !self.is_running() {
+            info!("process {} has already stopped", self.name());
+            return;
+        }
```

Comment:
nit: make this debug instead since it doesn't seem relevant to the user?

Reply:
done